packages/peregrine/lib/talons/Region/useRegion.js

```diff
@@ -48,16 +48,17 @@ export const useRegion = props => {
     }, [country, regionInputFieldApi, regionSelectFieldApi]);
 
     const { data, error, loading } = useQuery(getRegionsQuery, {
-        variables: { countryCode: country }
+        variables: { countryCode: country },
+        skip: !country
```

Comment:
Without this change, the `getRegions` query keeps failing when the app is loading at which point the `country` is `undefined`.

![image](https://user-images.githubusercontent.com/35203638/112070557-ecf8ed00-8b3b-11eb-8703-c3e486525141.png)